package.json

```diff
@@ -42,35 +41,29 @@
   "engines": {
     "node": ">= 18"
   },
-  "main": "./dist/cjs/src/index.js",
+  "main": "./dist/cjs/index.js",
   "module": "./dist/esm/src/index.js",
+  "types": "./dist/types/src/index.d.ts",
   "exports": {
     ".": {
       "import": "./dist/esm/src/index.js",
-      "require": "./dist/cjs/src/index.js",
+      "require": "./dist/cjs/index.js",
       "types": "./dist/types/src/index.d.ts"
     },
     "./stores": {
       "import": "./dist/esm/src/index-stores.js",
-      "require": "./dist/cjs/src/index-stores.js",
+      "require": "./dist/cjs/index-stores.js",
       "types": "./dist/types/src/index-stores.d.ts"
     },
     "./tests": {
       "import": "./dist/esm/tests/test-suite.js",
       "types": "./dist/types/tests/test-suite.d.ts"
     }
   },
-  "browser": {
-    "./dist/esm/src/index.js": "./dist/bundles/browser.js",
-    "./dist/cjs/src/index.js": "./dist/bundles/browser.js"
-  },
   "@comment react-native": [
     "directive for metro (react native build tool)."
   ],
-  "react-native": {
-    "./dist/esm/src/index.js": "./dist/esm/src/index.js",
-    "./dist/cjs/src/index.js": "./dist/esm/src/index.js"
-  },
+  "react-native": "./dist/cjs/index.js",
```

Comment:
Sanity: I thought the react-native folks had more success depending on the `esm`? Is `cjs` good for them to use again?

Reply:
for now! it seems to work better since the bundle now injects all of the third party dependencies that don't include cjs distributions or `react-native` import paths which prevents folks from having to remap anything themselves. previously they'd have to do [this](https://github.com/TBD54566975/web5-react-native-metro-config). not 100% sure this will end up holding up though! if it doesn't we'll switch back to esm